docs/blog/newsletter-august-2019.rst

```diff
@@ -0,0 +1,146 @@
+.. post:: August 2, 2019
+   :tags: newsletter
+
+#######################################
+Write the Docs Newsletter – August 2019
+#######################################
+
+Hello everyone! It's Beth and the team here, with another packed newsletter issue for you.
+
+A quick correction first: last month we said that the Australia conference is in Melbourne, when it is in fact in **Sydney**. Really sorry about that. If you'd like to speak in **Sydney**, the `CFP </conf/australia/2019/cfp/>`__ is still open: you have until 9th August to submit.
+
+In the other hemisphere, the `Prague speaker speaker schedule is up </conf/prague/2019/schedule/>`__, and `tickets are almost sold out </conf/prague/2019/news/events-activities/>`__. Once they're gone, they're gone, so if you'd like to come, buy your tickets ASAP!
+
+One last thing - we're delighted to have some new recruits to the newsletter team, which means we now have one more story about what the community has been talking about on Slack! Enjoy :) The newsletter team takes September off, so we'll be back in your inboxes come October.
+
+------------------------------------------------
+Adventures in generating docs from code comments
+------------------------------------------------
+
+A question about how to improve on tools like `Javadoc <https://www.oracle.com/technetwork/java/javase/documentation/index-jsp-135444.html>`__ and `doxygen <http://www.doxygen.nl/>`__ yielded many insights into getting great results with docs generated from code comments:
+
+* Javadoc is pretty standard in the Java world, so you can expect that Java devs are already looking for the Javadoc - and you can help them just by making sure it's easy to find.
```

Comment:
That is a lot of java on one line!

Reply:
I already removed a few :D will try to get it down a bit more!

Reply:
:D I guess I did overdo it

Reply:
It's super difficult with this topic!